### ISSUE TITLE
QuickPick in web with VoiceOver does not read anything

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#96578]
1. yarn web
2. VoiceOver
3. Cmd + P, use keyboard navigation to change focus -> VoiceOver does not rad anything

I can repro with Safari and Chrome.
Does not repro on Linux and Win. So might be a VoiceOver issue.

**Comment by bpasero** [comment]
I can reproduce in non-quick-open pickers too, so this seems a generic quick pick widget issue to me.

**Comment by isidorn** [comment]
Yes, a general quick pick widget issue correct.

**Comment by chrmarti** [comment]
VoiceOver does not handle the case where two different elements use `aria-activedescendant` to point to the same set of elements. In this case the list widget uses `aria-activedescendant` to announce the currently focused element and QuickPick's input box does the same.

The situation improves when only one of the two point to an element in the list at the same time (depending on which of the two has DOM focus). It then works until you `shift + tab` from the list widget back to the input box. At that point it seems to always reread what was last focused while the list widget had DOM focus.

I'm hesitant to check that in since it is unclear what other screenreaders will do with it.

@isidorn Any advice you can share?

**Comment by isidorn** [comment]
@chrmarti thanks for looking into this.
To clarify, you have elements A, B and C.
Elements A and B are pointing to C via `aria-activedscendent`?
So A -> C, B -> C
It is not A -> B -> C, correct?

I do not understand how an unfocused element could influence what VoiceOver is reading.
fyi @joanmarie on potential ideas (even though Orca works great)

**Comment by chrmarti** [comment]
Correct, it's A -> C, B -> C. A is the list, B is the input box and C is one of the list elements.

**Comment by isidorn** [comment]
@chmarti This sounds like a Chrome issue. If you can create a simple dummy web page and try to repro with Chrome and VoiceOver. I do not think it should be very complex.
I searched through the open Chromium issues I could not find anything.

On the other side I believe we should have some workaround for this.

**Comment by TylerLeonhardt** [comment]
This seems to only be an issue on Safari now.
